abb_irb6600_support/package.xml

```diff
@@ -0,0 +1,41 @@
+<package>
+  <name>abb_irb6600_support</name>
+  <version>0.2.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the ABB IRB 6600 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for ABB IRB 6600 manipulators.
+    </p>
+    <p>
+      Joint limits and max joint velocities are based on the information in the 
+      <a href="https://library.e.abb.com/public/f0a2a7361e593b9fc125766d003e9818/3HAC023933-001_revH_en_library.pdf">ABB 
+      IRB 6600 technical data sheet</a> (Version: 3HAC 023933-001 Rev. H, May 2004). All URDFs / XACROs are 
+      based on the default motion and joint velocity limits, unless noted otherwise (ie: no 
+      support for high speed joints, extended / limited motion ranges or other options).
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author email="levi.armstrong@swri.org">Levi Armstrong</author>
+  <maintainer email="levi.armstrong@swri.org">Levi Armstrong</maintainer>
+  <license>Apache2</license>
+
+  <url type="website">http://wiki.ros.org/abb</url>
```

Comment:
Sort-of: I meant: should we create a page for the `abb_experimental` repository? Whether we create pages for the various pkgs is then up to you/us. Just seems more correct to link user to `wiki.ros.org/abb_experimental`, instead of the main page, where they won't find these pkgs.

We could then also document how to 'install' the experimental pkgs on that page.